ui/src/components/settings/BasicSettings.vue

```diff
@@ -125,6 +125,7 @@
                     show-alpha
                 />
             </el-form-item>
+            <slot name="formItems"/>
```

Comment:
```suggestion
            <slot name="form-items"/>
```
please use kebab-case for slots' names